crates/sui-graphql-rpc/tests/snapshots/snapshot_tests__schema_sdl_export.snap

```diff
@@ -101,6 +101,24 @@ enum AddressTransactionBlockRelationship {
 	PAID
 }
 
+type AuthenticatorStateCreateTransaction {
+	"""
+	A workaround to define an empty variant of a GraphQL union.
+	"""
+	_: Boolean
+}
+
+type AuthenticatorStateExpireTransaction {
+	"""
+	Expire JWKs that have a lower epoch than this.
```

Comment:
Not sure, but is this supposed to be `expired` instead of `expire`?

Reply:
No it's the verb -- this transactions causes the JWKs that have a lower epoch than this to expire.

Reply:
Nice, thanks!